src/Amazon.Lambda.Tools/TemplateProcessor/TemplateProcessorManager.cs

```diff
@@ -301,6 +304,29 @@ private async Task<UpdateResourceResults> PackageDotnetProjectAsync(IUpdateResou
 
         }
 
+        private string[] RemoveProjectLocationArgument(string[] args)
+        {
+            List<string> argumentList;
```

Comment:
You don't need to declare here. L312 should just be 

var argumentList = new List<string>();